src/screen/addressScreen/editAddress.tsx

```diff
@@ -12,7 +13,7 @@ const EditAddress = ({navigation, route}) => {
         rightIcon={'close'}
         iconSize={28}
         onLeftIconPress={() => {
-          navigation.navigate('address');
+          RootNavigation.navigate('address');
```

Comment:
Here again, If it's a back event, then use pop method instead of again navigating to that screen

Reply:
used RootNavigation.pop() method for back functionality.